.stylelintrc

```diff
@@ -1,4 +1,7 @@
 {
+  "extends": [
```

Comment:
I see we just extend the config from prettier. This means that we don't have any conflicts like in tslint?